quantum/template/avr/config.h

```diff
@@ -23,8 +23,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define VENDOR_ID 0xFEED
 #define PRODUCT_ID 0x0000
 #define DEVICE_VER 0x0001
-#define MANUFACTURER %YOUR_NAME%
-#define PRODUCT %KEYBOARD%
+#define MANUFACTURER % YOUR_NAME %
```

Comment:
Please undo the changes to the template files - clang-format adding the spaces here breaks the new_keyboard script. The LTO changes are OK though.

Reply:
will do